aptos-move/block-executor/src/captured_reads.rs

```diff
@@ -150,6 +155,115 @@ struct GroupRead<T: Transaction> {
     inner_reads: HashMap<T::Tag, DataRead<T::Value>>,
 }
 
+/// Defines different ways `AggregatorResolver` can be used to read its value
+/// from the state.
+/// The enum variants should not be re-ordered, as it defines a relation
+/// Bounded < Value
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub enum DelayedFieldReadKind {
+    /// The returned value is guaranteed to be correct.
+    Bounded,
```

Comment:
Wondering about the terminology here. Why not call it "LastCommittedValue" and "AggregatedValue" as before? Just based on the terms "Bounded" and "Value", it isn't clear what the variant means.